src/Microsoft.DotNet.Github.IssueLabeler/GithubIssueDownloader.cs

```
@@ -70,9 +86,9 @@ public async Task DownloadAndSaveAsync()
                         sb.Clear();
                     }
                 }
-                catch (Exception)
+                catch (Exception ex)
                 {
-                    Console.WriteLine($"Issue {i} does not exist");
+                    Console.WriteLine($"Issue {i}: " + ex.Message);
```

Exception may happen when we exceed rate limit as well. If it is in fact because the issue does not exist then ex,Message is "Not Found". so with this change we are not loosing information

---

can we throw an exception and stopped the app if api rate limit is exceeded.